src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -1076,6 +1076,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('cap_mbps', arg_group='Additional Flags', help="Caps the transfer rate, in megabits per second. "
                    "Moment-by-moment throughput might vary slightly from the cap. "
                    "If this option is set to zero, or it is omitted, the throughput isn't capped. ")
+        c.positional('extra_options', nargs='*', is_experimental=True, default=[],
```

Comment:
I think `[]` is already the default:

```py
import argparse

parser = argparse.ArgumentParser()
parser.add_argument('extra', nargs="*")

print(parser.parse_args(''.split()))

>>> Namespace(extra=[])
```

Reply:
I didn't know and added manually. Thanks for notifying.